rest/admin_api.go

```diff
@@ -403,6 +403,21 @@ func (h *handler) handleGetRawDoc() error {
 		return err
 	}
 
+	metaMap, err := doc.GetMetaMap(h.db.Options.UserXattrKey)
+	if err != nil {
+		return err
+	}
+
+	metaMapBytes, err := base.JSONMarshal(metaMap)
+	if err != nil {
+		return err
+	}
+
+	rawBytes, err = base.InjectJSONPropertiesFromBytes(rawBytes, base.KVPairBytes{Key: "_meta", Val: metaMapBytes})
+	if err != nil {
+		return err
+	}
```

Comment:
Nitpick: There's a variant of this util that does the marshalling for you. But don't bother changing unless there's other stuff to do.

```suggestion
	rawBytes, err = base.InjectJSONProperties(rawBytes, base.KVPair{Key: "_meta", Val: metaMap})
	if err != nil {
		return err
	}
```